pkg/plugin/work-in-progress/plugin/event_handler_unit_test.go

```diff
@@ -1,12 +1,11 @@
 package plugin_test
 
 import (
-	. "github.com/arquillian/ike-prow-plugins/pkg/internal/test"
 	"github.com/google/go-github/github"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
-	gock "gopkg.in/h2non/gock.v1"
+	"gopkg.in/h2non/gock.v1"
```

Comment:
that should not be modified, but as the `goimports` doesn't complain about it then I wouldn't care...

Reply:
I think it skips the test packages